rockspec/rxlua-scm-2.rockspec

```diff
@@ -0,0 +1,24 @@
+package = "rxlua"
+version = "scm-2"
+
+source = { url = "git://github.com/bjornbytes/RxLua.git" }
+
+description =
+{
+  summary = "Reactive Extensions for Lua",
+  homepage = "https://github.com/bjornbytes/RxLua/tree/master",
+  license = "MIT/X11",
+  maintainer = "tie.372@gmail.com",
+  detailed = [[
+RxLua gives Lua the power of Observables, which are data structures that represent a stream of values that arrive over time. They're very handy when dealing with events, streams of data, asynchronous requests, and concurrency.
+]]
+}
+
+dependencies = { "lua >= 5.1" }
```

Comment:
I wasn't sure whether to make the changes retroactive or not. It didn't seem like rockspec of other lua projects did that. But you can certainly add that change in, naming it like `rxlua-0.0.1-2.rockspec` and `rxlua-0.0.2-2.rockspec`.